src/mappings/entities/fundingEvent.ts

```diff
@@ -0,0 +1,29 @@
+import { SubstrateEvent } from "@subql/types";
+import { FundingEvent } from "../../types";
+import { Codec } from "@polkadot/types/types";
+import { hexToAscii } from "../util";
+
+export async function handleFundingEvent(
+  block_id: number,
+  event_id: string,
+  module_id: string,
+  params: Codec[],
+  event: SubstrateEvent
+): Promise<void> {
+  if (module_id === "asset" && event_id === "Issued") {
+    const rawTicker = params[1];
+    const ticker = hexToAscii(rawTicker.toString());
```

Comment:
the way I do it is: `removeNullChars(u8aToString(item.toU8a()))` could you perhaps create a util function serializeTicker that is shared here and in `serializeLikeHarvester.ts` please?